services/brig/brig.integration.yaml

```diff
@@ -228,6 +228,10 @@ optSettings:
   setOAuthEnabled: true
   setOAuthRefreshTokenExpirationTimeSecs: 14515200 # 24 weeks
   setOAuthMaxActiveRefreshTokens: 10
+  setPasswordHashingOptions:
+    iterations: 1
+    memory: 131072 # 2 ^ 17 kibibytes
```

Comment:
maybe add a section to docs.wire.com where you name this stuff, explain "kibibyte", and reference argon2id rfc, maybe other stuff of relevance.